docs/src/touch-events.md

```diff
@@ -657,4 +657,4 @@ with sync_playwright() as p:
     page = context.new_page()
     test_pinch_in_gesture_to_zoom_out_the_map(page)
     browser.close()
-```
\ No newline at end of file
+```
```

Comment:
I can't understand what has changed in this line. Perhaps revert?

Reply:
Usually github shows "no empty line at end of file" icon. However, there is no such icon on either side, so I am a bit confused.